external/upstream/libxc/CMakeLists.txt

```diff
@@ -15,9 +15,9 @@ else()
         # Default: use a stable release tarball of libxc. To use the
         # development version of libxc, instead, comment the URL line,
         # and uncomment the GIT lines.
-        URL http://www.tddft.org/programs/octopus/down.php?file=libxc/4.3.4/libxc-4.3.4.tar.gz
-        #GIT_REPOSITORY https://gitlab.com/libxc/libxc.git
-        #GIT_TAG master
+        #URL http://www.tddft.org/programs/octopus/down.php?file=libxc/4.3.4/libxc-4.3.4.tar.gz
+        GIT_REPOSITORY https://gitlab.com/libxc/libxc.git
+        GIT_TAG 4.3.4
```

Comment:
actually, keep using `URL` and keep the `GIT_`* ones commented. I think there's a comparable interface in GL to the GH URL lines for other externals. lgtm can't build from `GIT_`, only from `URL`.

Reply:
Ah, didn't realize that.  Will change it now..